examples/tv-app/android/java/KeypadInputManager.h

```diff
@@ -41,5 +41,5 @@ class KeypadInputManager : public KeypadInputDelegate
     jmethodID mSendKeyMethod          = nullptr;
 
     // TODO: set this based upon meta data from app
-    uint32_t mDynamicEndpointFeatureMap = 7;
+    static constexpr uint32_t kEndpointFeatureMap = 7;
```

Comment:
could this be computed from some OR-ing of constant values?  https://github.com/project-chip/connectedhomeip/blob/master/src/controller/data_model/controller-clusters.matter#L8216 seems to define the Feature.

Worst-case this should at least comment that we have kNavigationKeyCodes, kLocationKeys and kNumberKeys.

I would very much prefer to read something like:

```
   BitMask(
       KeypadInputManager::Feature::kNavgationKeyCodes,
       KeypadInputManager::Feature::kLocationKeys,
       KeypadInputManager::Feature::kNumberKeys,
   ).Raw()
```

however not sure what can be done in a constexpr here.

Reply:
Added it to the issue:
https://github.com/project-chip/connectedhomeip/issues/31991